learn/os_setup/install_gfortran.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: book
+title: Installing GFortran
+permalink: /learn/os_setup/install_gfortran
+---
+
+GFortran is the name of the [GNU Fortran project](https://gcc.gnu.org/fortran/). The main [wiki](https://gcc.gnu.org/wiki/GFortran) page offers many helpful and informational links about GFortran, as well as Fortran in general. In this guide, the installation process for GFortran on Windows, Linux, and MacOS is presented in a beginner-friendly format based on information from [GFortranBinaries](https://gcc.gnu.org/wiki/GFortranBinaries).
+
+## Windows
+
+In Windows, there are two main *vendors* that offer builds of GFortran, namely the [MinGW](http://www.mingw.org/) and the [Cygwin](https://cygwin.com/index.html). Directly derived from MinGW is the [Mingw-w64](http://mingw-w64.org/) project that offers 64-bit executables for many platforms. 
+
+As of this date, there are three sources that provide quick and easy to install GFortran compiler under Windows:
+1. [http://www.equation.com](http://www.equation.com/servlet/equation.cmd?fa=fortran), provides x32 and x64 executables of the latest (10.2) gcc-version.
+2. [TDM GCC](https://jmeubank.github.io/tdm-gcc/articles/2020-03/9.2.0-release), provides x32 and x64 executables of the 9.2 gcc-version.
+3. [Mingw-w64](http://mingw-w64.org/doku.php/download/mingw-builds) provides x64 executable of the 7.2 gcc-version.
```

Comment:
Luckily there is a new distribution that provides version 10.2.

There is a tight relation between MSYS2 and MinGW-w64, this is not quite clear from the text, but I agree that it is pretty difficult to describe it. I struggle with it myself all the time. This bit should be rewritten. No critique, mind you, it is just that the current text is not sufficiently precise and might cause the same confusion as I have had (or still have).

Reply:
I added further text briefly describing MSYS2, Cygwin and WSL and their differences so I will mark this discussion point as resolved.